src/applications/vaos/tests/new-appointment/components/ReviewPage/index.direct.unit.spec.jsx

```diff
@@ -223,13 +223,15 @@ describe('VAOS <ReviewPage> direct scheduling', () => {
       'Something went wrong when we tried to submit your appointment. Call your VA medical center to schedule this appointment.',
     );
 
-    await screen.findByText('307-778-7550');
+    // await screen.findByText('307-778-7550');
```

Comment:
Not used anymore?

Reply:
Good catch. That line can be removed all together because the check for facility phone is replaced in line 233.